toolbox/utils.py

```diff
@@ -442,6 +465,259 @@ def as_command_choices(*args: t.Any, **kwargs: t.Any) -> t.Sequence[hikari.Comma
     return _list_to_command_choices(choices)
 
 
+class MarkdownFormat(IntFlag):
+    NONE = 0
+    STRIKETHROUGH = 1
+    ITALIC = 2
+    BOLD = 4
+    UNDERLINE = 8
+    CODE_BLOCK = 16
+    MULTI_CODE_BLOCK = 32
+    QUOTE = 64
+    MULTI_QUOTE = 128
+    SPOILER = 256
+
+    ALL = STRIKETHROUGH | ITALIC | BOLD | UNDERLINE | CODE_BLOCK | MULTI_CODE_BLOCK | QUOTE | MULTI_QUOTE | SPOILER
+
+
+def remove_markdown(content: str) -> str:
+    """Removes the markdown formatting from discord messages.
+
+    Parameters
+    ----------
+    content : str
+        The `str` object, which needs their content cleaned from discord's markdown formatting.
+
+    Returns
+    -------
+    str
+        The `cleaned` string without markdown formatting.
```

Comment:
"cleaned" does not need backticks around it. Applies to all functions.

Reply:
Moving to a new file is a great idea, had the same halfway in.

Is pluralisation a common thing? My thought was that it is a format, which is singular.

"cleaned" is in backticks because it refers to the variable name, thought that is how it is done.

Do you have a suggestion how to apply the enum? I only used them a few times, but never made one myself.

Thanks for the feedback!

Reply:
For "cleaned", that variable name would never be exposed to the user anyways, since it is the return value, so it is irrelevant.

Reply:
I applied your suggestions.

Another thing that could be changed is the removal of styling specified at the beginning of multiline codeblocks.
I already have an idea how to implement.

Still would need some assistance with the enum.

Reply:
I'm not sure what assistance you're referring to exactly. I'm not sure the enum needs to be used in this context -- what were you planning for it?

Reply:
Nvm Hyper filled me in. It would involve bitwise operations, but I can't quite remember exactly how you need to implement them. Hyper probs knows.